### DISCUSSION TITLE
Uno and resharper not playing along

### DISCUSSION THREAD

**Discussion by digger73** [unoplatform/uno#12576] [Q&A]
Hi
When creating a new project using the default template resharper will complain about un-reasoalvable symbols in the MainPage.xaml. Letting resharper resolve this, results in a project that fails when compiled. The two errors are:
"Unable to resolve navigationBar" and "Unable to resolve "AutomationProperties". Resolving these resharper will insert "Uno.Toolkit.UI.NavigationBar" and "xmlns:automation="clr-namespace:Microsoft.UI.Xaml.Automation;assembly=Uno.UI"

We would of course like to continue using resharper along with implementing in UNO, are anyone aware of the reason why I see this, and maybe resolve it so resharper doesn't complain about these not missing resources.

**Comment by dr1rrb** [comment]
Hey @digger73, thanks for the repport.

For the compilation issue, which options did you choose in the wizard? On my side when I create an app using the "blank" option I don't have any compilation issue. @dansiegel might have more idea on this?

For ReSharper namespace resolution (and globally to have better suggestions), you should make sure to select the "windows" target in drop down at the top left of the document view:
![image](https://github.com/unoplatform/uno/assets/8635919/88a13c47-3729-437b-b2eb-7b3e38841000)
Doing this, when I press alt+enter and request to ReSharper to add the namespace for `<NavigationBar` it will add ` xmlns:ui="using:Uno.Toolkit.UI"`

**Reply by dansiegel** [reply]
Technically speaking `xmlns:automation="clr-namespace:Microsoft.UI.Xaml.Automation;assembly=Uno.UI"` should work just fine if (and only if) you target Uno Platform exclusively. If you are building WinUI this will not work as the Assembly would be different. If you instead have the `xmlns` set like @dr1rrb showed `using:Microsoft.UI.Xaml.Automation` that would work across all targets. That said it seems weird since that should come in from the default namespace.

For other controls like those in the Uno.Toolkit then it wouldn't matter which way you add the `xmlns` since those assemblies will be the same.

**Reply by agneszitte** [reply]
(cc @ArchieCoder for the related details)

**Comment by ArchieCoder** [comment]
Thank you all, not only selecting the platform via that dropdown fixed the issue, it also fixes the issue that I had with hot reload.

Since the day I started to play with Uno, I was under the impression the platform change was only via this menu. I didn't change/"see" the dropdown in the top status bar.

![image](https://github.com/unoplatform/uno/assets/1608424/55c81e20-6ada-440d-84c0-232f058031bc)
